docs/docsite/rst/scenario_guides/cloud_guides.rst

```diff
@@ -13,13 +13,8 @@ Guides for using public clouds are moving into collections. We are migrating the
 .. toctree::
    :maxdepth: 1
 
-   guide_alicloud
    guide_cloudstack
    guide_gce
    guide_azure
    guide_online
```

Comment:
The online modules and inventory plugin also live in community.general, I've added the Online guide to https://github.com/ansible-collections/community.general/pull/7714.